src/hotspot/cpu/ppc/macroAssembler_ppc.hpp

```diff
@@ -54,6 +54,8 @@ class MacroAssembler: public Assembler {
   inline static int largeoffset_si16_si16_hi(int si31) { return (si31 + (1<<15)) >> 16; }
   inline static int largeoffset_si16_si16_lo(int si31) { return si31 - (((si31 + (1<<15)) >> 16) << 16); }
 
+  // Branch-free implementation to convert !=0 to 1.
+  void normalize_bool(Register dst, Register temp, bool use_64bit);
```

Comment:
@TheRealMDoerr  I have placed it here based on the comment   //
  // Optimized instruction emitters
  //

I think that is more relevant to this. Let me know if otherwise

Reply:
I'd place them below `set_cmpu3` which is a similar function.